htlcswitch/link.go

```diff
@@ -1700,6 +1747,15 @@ func (l *channelLink) handleDownstreamUpdateAdd(pkt *htlcPacket) error {
 //
 // TODO(roasbeef): add sync ntfn to ensure switch always has consistent view?
 func (l *channelLink) handleDownstreamPkt(pkt *htlcPacket) {
+	if pkt.htlc.MsgType().IsChannelUpdate() &&
+		!l.quiescer.canRecvUpdates() {
+
+		l.log.Warnf("unable to process channel update. "+
```

Comment:
If we can't handle this, then we need to bounce it. That way it's removed from our mailbox, and also we send a cancel back in the incoming direction (can't service it on this link rn). Alternatively we could try to introduce the old QoS queue, but that would be a distinct design iteration.

Reply:
Or is the intent that it sits in the mailbox, to be retransmitted once we reconnect, or the `stfu` session has concluded?

Reply:
yes, we need to retransmit when quiescence resolves. The expected end-to-end resolution time is on the order of <1s. We could bounce it if you prefer, as it gives the sender an opportunity to try a different path if they're really impatient but I don't think it's *strictly* necessary given the anticipated patterns of usage here.